src/main/drivers/light_ws2811strip.c

```diff
@@ -60,7 +60,11 @@
 #define WS2811_DMA_BUF_CACHE_ALIGN_LENGTH (WS2811_DMA_BUF_CACHE_ALIGN_BYTES / sizeof(uint32_t))
 DMA_RW_AXI __attribute__((aligned(32))) uint32_t ledStripDMABuffer[WS2811_DMA_BUF_CACHE_ALIGN_LENGTH];
 #else
+#if defined(STM32F7)
```

Comment:
As we no longer accept specific MCU defines need to add a define to STM32F7 target like `USE_LED_STRIP_FAST_DATA` instead.

Reply:
There is already existing define to define variable in DMA-accessible memory.